infrastructure/src/main/java/org/corfudb/infrastructure/BatchProcessor.java

```diff
@@ -194,8 +194,8 @@ private void processor() {
                                 log.warn("Unknown BatchWriterOperation {}", currOp);
                         }
                     } catch (Exception e) {
-                        log.error("Stream log error. Batch [queue size={}]. StreamLog: [trim mark: {}, tails: {}].",
-                                operationsQueue.size(), streamLog.getTrimMark(), streamLog.getAllTails(), e);
+                        log.debug("Stream log error. Batch [queue size={}]. StreamLog: [trim mark: {}].",
```

Comment:
The title of this patch suggests we want to reduce the occurrence of OverwriteException in logs, but here you reduced log level for all kinds of exceptions. It swallows important exceptions like IOException if we are in production which may use higher log level. Why don't we check the exception type and process overwriteException differently?

Reply:
I don't see any reasons to do that, what's the motivation?
 anyway we are going to write this message in the log, just without stream tails

Reply:
And also the exception is not swallowed, it goes to the CompletableFuture and can (will?) be handled on next steps

Reply:
In production, we may use INFO level logging. Then since you make it debug level, we will miss  possibly some really critical but unexpected exception.

Reply:
we don't use it. If we migrate to INFO level, then yes. But the migration is a huge change. When it happens, this also will be changed. Now I vote for - keep it consistent with the current implementation. And also this must be the debug level anyway because we handle the exception on next staeps

Reply:
Why is this related to migration? It's just a deployment configuration right?

Reply:
I don't understand your question, sorry

Reply:
Changed, according to offline discussion

Reply:
Cool.